pkg/spec/config_linux.go

```diff
@@ -98,6 +99,26 @@ func addDevice(g *generate.Generator, device string) error {
 	if err != nil {
 		return errors.Wrapf(err, "%s is not a valid device", src)
 	}
+	if rootless.IsRootless() {
+		if _, err := os.Stat(src); err != nil {
+			if os.IsNotExist(err) {
+				return errors.Wrapf(err, "the specified device %s doesn't exist", src)
+			}
+			return errors.Wrapf(err, "stat device %s exist", src)
+		}
+		perm := "ro"
+		if strings.Contains(permissions, "w") {
+			perm = "rw"
```

Comment:
Maybe we can add a log here? It makes sense to overwrite the permissions but it might surprise some users.

Reply:
it is used to convert from the "rwm" format to either "ro" or "rw".  If you'd like I can add a comment.  There are still tests failures so I'll probably need to repush

Reply:
If you need to repush, it would be nice to have a log. Otherwise, don't bother adding it.

Reply:
I am not sure there is need to log the action.  We are just converting from one format (the "rwm" format for devices) to another (either "ro" or "rw" bind mount), but that should be transparent to the user